libs/wingsdk/src/fs/fs.ts

```diff
@@ -0,0 +1,302 @@
+import * as fs from "fs";
+import * as os from "os";
+import * as path from "path";
+import * as yaml from "js-yaml";
+import { InflightClient } from "../core";
+import { Json } from "../std";
+
+/**
+ * Convert the path to Windows format if running on Windows.
+ * @param p The path to convert.
+ * @returns The Normalized path
+ */
+function localePath(p: string): string {
+  if (process.platform !== "win32") {
+    return p;
+  }
+  let winPath = p.split(path.posix.sep).join(path.win32.sep);
+  // If the path starts from the root directory, the first segment represents a specific drive.
+  winPath = winPath.replace(/^\\([A-Za-z])\\/, "$1:\\");
+  return winPath;
+}
+
+/**
+ * Convert the path to POSIX format.
+ * @param p The path to convert.
+ * @returns The Normalized path.
+ */
+function posixPath(p: string): string {
```

Comment:
It's not really possible to convert windows path to posix path.

`C:\` is not that same as `/c/`, nor is it always the same as `/`.

If you're running on windows and you attempt to read `/`, I think a reasonable behavior of wing would be to just let windows handle it. Which means it'll read the default system drive afaik. If we do that then I don't think this function is needed.

Reply:
Do you mean we should keep the drive in the posix-like path?

Reply:
> Do you mean we should keep the drive in the posix-like path?

Yes, I think no attempt should be made convert absolute Windows paths into a _truly_ posix-compliant path. Converting `\` to `/` is useful since it helps with consistency across platforms and doesn't hurt functionality, but mangling the drive letter is not possible to handle automatically without possibly breaking something.